packages/transitive-overlay/src/index.tsx

```diff
@@ -199,6 +201,10 @@ const TransitiveCanvasOverlay = ({
     zoomToGeoJSON(polyline.toGeoJSON(activeLeg.legGeometry.points));
   }, [activeLeg]);
 
+  if (!transitiveData) return <></>;
```

Comment:
Would it be possible to just put this at the top, along with anything that uses `activeLeg`, so that it's not necessary to have the big ternary above?

Reply:
Good question! Unfortunately, [all hooks have to be rendered outside conditions, before `return` etc](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level).